sale_product_set/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Anybox
+# Copyright 2018 Camptocamp, ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Sale product set',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'author': 'Anybox, Odoo Community Association (OCA)',
+    'version': '12.0.1.0.0',
+    'website': 'https://github.com/OCA/sale-workflow',
+    'summary': "Sale product set",
+    'depends': [
+        'sale',
+    ],
+    'data': [
```

Comment:
Sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

Reply:
done